src/libImaging/Resample.c

```diff
@@ -286,88 +286,30 @@ void
 ImagingResampleHorizontal_8bpc(
     Imaging imOut, Imaging imIn, int offset, int ksize, int *bounds, double *prekk) {
     ImagingSectionCookie cookie;
-    int ss0, ss1, ss2, ss3;
-    int xx, yy, x, xmin, xmax;
-    INT32 *k, *kk;
+    int xx, yy, b, x, xmin, xmax, ss;
+    UINT8 *out, *in;
+    INT32 *k;
 
     // use the same buffer for normalized coefficients
-    kk = (INT32 *)prekk;
+    const INT32 *kk = (INT32 *)prekk;
```

Comment:
```suggestion
    kk = (INT32 *)prekk;
```
Compiling, I get `warning: initializing 'unsigned char *' with an expression of type 'const unsigned char *' discards qualifiers`.
This fixes it.

Reply:
I don't see that warning, but I made `k` `const` as well which should also fix this issue.